src/MagicWormhole/Internal/Dispatch.hs

```diff
@@ -0,0 +1,235 @@
+{-# LANGUAGE DeriveAnyClass #-}
```

Comment:
Fundamentally unsure about whether separating "dispatch" (corralling RPCs etc) from websocket code is a good idea